### PR TITLE
Pushing PVC create and delete results into performance dashboard

### PR DESCRIPTION
**Pull request by Avilir**
This PR is writing all he elasticsearch links (for each individual test from parametrize) to the results into single file, and then push them into the performance dashboard (only if all tests finished OK)

Signed-off-by: Avi Layani <alayani@redhat.com>

### CODE REVIEW

**Review by ypersky1980 — CHANGES_REQUESTED** [review]
@Avilir please use class ResultsAnalyse from ocs_ci.ocs.perfresult . 

from ocs_ci.ocs.perfresult import ResultsAnalyse 

and pass test index name to it. 

After you use it - please remove ResultsAnalyse from this test.

**Comment by Avilir** [comment]
@ypersky1980 , changes done. thanks.

**Review by ypersky1980 — APPROVED** [review]

**Comment by keemano** [comment]
LGTM

**Comment by Avilir** [comment]
from the test log : 
```
2021-12-28 12:29:37  10:29:37 - MainThread - ocs_ci.utility.perf_dash.dashboard_api - INFO - Test results pushed to the DB!
2021-12-28 12:29:37  10:29:37 - MainThread - ocs_ci.ocs.perftests - INFO - Full results report can be found at : http://10.0.78.167:8080/index.php?version1=14&build1=24&platform1=1&az_topology1=1&test_name%5B%5D=3&submit=Choose+options
2021-12-28 12:29:37  10:29:37 - MainThread - ocs_ci.utility.perf_dash.dashboard_api - INFO - Closing the DB connection
2021-12-28 12:29:37  PASSED                                                                   [ 76%]
```

All the results are in the performance dashboard.

File: tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py [hunk 1]
```diff
@@ -294,8 +275,34 @@ def test_pvc_creation_deletion_measurement_performance(
...
+            cname=self, fname="test_pvc_creation_deletion_measurement_performance"
+        )
+        self.results_file = os.path.join(self.results_path, "all_results.txt")
+        log.info(f"Check results in {self.results_file}")
+        self.number_of_tests = 3
+        log.info("Check results for 'performance_extended' marker (3 tests)")
+        try:
+            self.check_tests_results()
+        except ex.BenchmarkTestFailed:
+            log.info("Look like performance_extended was not triggered")
```
Comment (ypersky1980) [inline tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py:301]: Spelling fix : Looks like, not Look like

Comment (Avilir) [reply tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py:301]: will fix in follow up PR's (more functionality - not related to dashboard - improvements need to be done in this TC)

**Review by ramkiperiy — APPROVED** [review]

**Review by ypersky1980 — APPROVED** [review]

**Pull request merged**